docs/NetworkPeeringForScalarDLAuditor.md

```diff
@@ -0,0 +1,62 @@
+# Create network peering for Scalar DL Auditor mode
+
+This document explains how to connect several private networks for ScalarDL Auditor mode. To make ScalarDL Auditor mode work properly, ScalarDL Ledger and ScalarDL Auditor need to connect with each other. 
+
+## Private network you must connect
+
+Depending on your application (client) deployment, you must connect several private networks.
+
+### Connect two private networks
```

Comment:
I think we should not do this pattern and recommend users have 3 private networks for Ledger, Auditor, and clients (applications) for correctness.
Let's talk about the recommended network architecture and explain the way to create it.

Reply:
Thank you for your review!

> I think we should not do this pattern and recommend users have 3 private networks for Ledger, Auditor, and clients (applications) for correctness.

I see. That's right. Indeed, users should have three private networks to separate administrative domains.

> Let's talk about the recommended network architecture and explain the way to create it.

I will consider the recommended network architecture and explanation. Please let me discuss this with you later.